models/twilio__account_overview.sql

```diff
@@ -7,6 +7,12 @@ with messages as (
     from {{ ref('int_twilio__messages')}}
 ),
 
+account_history as (
+
+    select * 
+    from {{ var('account_history')}}
+),
```

Comment:
I noticed we didn't bring in the account history staging model to the account overview model. I brought it in to add some complimentary dimensions.